code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -1053,17 +1054,12 @@ It'll return null if the organ doesn't correspond, so include null checks when u
 /**
   * Species-specific runechat colour handler
   *
-  * Checks the species datum flags and returns the appropriate colour
-  * Can be overridden on subtypes to short-circuit these checks (Example: Grey colour is eye colour)
+  * Returns the mob's runechat color which can be changed at a mirror or in preferences.
   * Arguments:
   * * H - The human who this DNA belongs to
   */
 /datum/species/proc/get_species_runechat_color(mob/living/carbon/human/H)
-	if(bodyflags & HAS_SKIN_COLOR)
-		return H.skin_colour
-	else
-		var/obj/item/organ/external/head/HD = H.get_organ("head")
-		return HD?.hair_colour
+	return H.runechat_color
```

Comment:
This proc is useless, make the caller just refer to this var

Reply:
this was a left over i forgot to remove.

Reply:
Completely axed, resolving as it is outdated.